src/SessionSetup/include/SessionSetup/SessionSetupUtils.h

```diff
@@ -10,13 +10,18 @@
 #include <memory>
 #include <optional>
 
+#include "ClientData/ProcessData.h"
+#include "GrpcProtos/process.pb.h"
 #include "OrbitGgp/SshInfo.h"
 #include "OrbitSsh/Credentials.h"
 
 namespace orbit_session_setup {
 
 [[nodiscard]] orbit_ssh::Credentials CredentialsFromSshInfo(const orbit_ggp::SshInfo& ssh_info);
 [[nodiscard]] std::shared_ptr<grpc::Channel> CreateGrpcChannel(uint16_t port);
+[[nodiscard]] std::unique_ptr<orbit_client_data::ProcessData> TryToFindProcessData(
+    const std::vector<orbit_grpc_protos::ProcessInfo>& process_list,
+    const std::string process_name_or_path);
```

Comment:
You probably meant `const std::string&`. You could also consider a `string_view`.

Reply:
Yes I did, good catch.